src/authentification/authentification.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { UserService } from 'src/services/user/user.service';
+import * as bcrypt from 'bcrypt';
+import * as jwt from 'jsonwebtoken';
+import { User } from 'src/domain/models/user/user.model';
+require('dotenv').config();
+
+
+@Injectable()
+export class AuthentificationService {
+
+    constructor(
+      private usersService: UserService,
+      private jwtService: JwtService,
+    ){};
+
+
+    async login(email: string, password: string) {
+      try {
+
+        const user = await this.usersService.getUserByEmail(email); 
+        
+        const userPassword = user.getPassword();    
+        const pass = await bcrypt.compare(password, userPassword)
+        
+        if (pass) { 
```

Comment:
`pass` n'est pas un nommage claire. En général un booléen est préfixé par `is` ou `has`. Donc ici on s'attendrai à `isPasswordsMatch` par exemple.